Buildings/Experimental/DHC/Examples/Combined/SeriesVariableFlowAgentControl.mo

```diff
@@ -0,0 +1,266 @@
+          {-80,142},{-20,142}}, color={0,127,255}));
+  connect(dis.port_bDisSup, TDisWatRet1.port_a)
+    annotation (Line(points={{20,142},{80,142},{80,138}}, color={0,127,255}));
+  connect(dis.ports_bCon, bui.port_aSerAmb) annotation (Line(points={{-12,152},
+          {-14,152},{-14,180},{-10,180}},color={0,127,255}));
+  connect(dis.ports_aCon, bui.port_bSerAmb) annotation (Line(points={{12,152},{
+          16,152},{16,180},{10,180}},
+                                   color={0,127,255}));
+  connect(dis.TOut, conPum.TMix) annotation (Line(points={{22,136},{34,136},{34,
+          150},{-272,150},{-272,134},{-382,134},{-382,-168},{-54.0308,-168},{
+          -54.0308,-167.4}},                        color={0,0,127}));
+  connect(TDisWatSup.port_b, supDisPluFlo.port_a)
+    annotation (Line(points={{-80,30},{-80,82}}, color={0,127,255}));
+  connect(supDisPluFlo.port_b, TDisWatSup1.port_a) annotation (Line(points={{-80,102},
+          {-80,118}},                color={0,127,255}));
+  connect(TDisWatRet1.port_b, retDisPluFlo.port_a)
+    annotation (Line(points={{80,118},{80,102}}, color={0,127,255}));
+  connect(retDisPluFlo.port_b, TDisWatRet.port_a) annotation (Line(points={{80,82},
+          {80,10}},                 color={0,127,255}));
+  connect(pipeGroundCouplingMulti[1:(nBui+1)].heatPorts[1], dis.heatPorts)
+    annotation (Line(points={{0,95},{0,96},{0.4,96},{0.4,139.8}},
+        color={127,0,0}));
+  connect(supDisPluFlo.heatPort, pipeGroundCouplingMulti[nBui + 2].heatPorts[1])
+    annotation (Line(points={{-70,92},{0,92},{0,95}},
+        color={191,0,0}));
+  connect(retDisPluFlo.heatPort, pipeGroundCouplingMulti[nBui + 3].heatPorts[1])
+    annotation (Line(points={{70,92},{0,92},{0,95}},                     color={
+          191,0,0}));
+  connect(HXtemperature.y[1], pla.TSewWat) annotation (Line(points={{-267,40},{
+          -172,40},{-172,7.33333},{-161.333,7.33333}}, color={0,0,127}));
+  connect(HXtemperature.y[1], pumPlantControlNsew.TSou) annotation (Line(points={{-267,40},
+          {-260,40},{-260,20},{-288,20},{-288,3},{-281.538,3}},
+                     color={0,0,127}));
+  annotation (
+  Diagram(
+  coordinateSystem(preserveAspectRatio=false, extent={{-400,-260},{360,260}})),
+    __Dymola_Commands(
+  file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/DHC/Examples/Combined/SeriesVariableFlowAgentControl.mos"
+  "Simulate and plot"),
+  experiment(
+      StopTime=604800,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Dassl"),
+    Documentation(revisions="<html>
+<ul>
+<li>
+January 20, 2023, by Ettore Zanetti:<br/>
+First implementation.
+</li>
+</ul>
+</html>", info="<html>
+<p>
+This model is identical to
+<a href=\"modelica://Buildings.Experimental.DHC.Examples.Combined.SeriesConstantFlow\">
+Buildings.Experimental.DHC.Examples.Combined.SeriesConstantFlow</a>
+except for the pipe diameter and the control of the main circulation pump.
```

Comment:
This documentation is copied from Buildings.Experimental.DHC.Examples.Combined.SeriesVariableFlow and needs to be updated.

Reply:
Updated documentation